build.gradle

```diff
@@ -792,6 +826,16 @@ distZip {
   }
 }
 
+task distRpm {
+  dependsOn product
+  dependsOn buildRpm
+}
+
+task distDeb {
+  dependsOn product
+  dependsOn buildDeb
+}
+
```

Comment:
done, but i had to add it to the base `buildRpm` and `buildDeb` targets instead, else build was failing with error like `classifier` method not found.